src/sessionDurationsToMinutes.ts

```diff
@@ -0,0 +1,16 @@
+function hoursToMinutes(hours: number): number {
+    return Math.round(hours * 60)
+}
+
+export const sessionDurationsToMinutes = (
```

Comment:
This is the new function that converts `session_lifetime` and `idle_session_lifetime` into their minute counterparts. 

It is most easily consumed when destructuring the result as can be seen in the handlers.